resources/views/metadata-tags.blade.php

```diff
@@ -4,11 +4,11 @@
     'image' => null,
 ])
 
-<meta name="description" content="{{ trim($description) }}" />
+<meta name="description" content="{!! trim(htmlentities($description)) !!}" />
 
-<meta property="og:title" content="{{ trim($title) }}" />
-<meta property="og:description" content="{{ trim($description) }}" />
+<meta property="og:title" content="{!! trim(htmlentities($title)) !!}" />
+<meta property="og:description" content="{!! trim(htmlentities($description)) !!}" />
```

Comment:
so basically this turns `'` into `&#039;` and then the `!!` makes it end up as `'` again in the eventual HTML so the meta property has `'`? 🤔

Reply:
with the changes on msq the text comes unencoded (so the `'` comes at it is). Once here the `{{ }}` directive is the one that used to encode the text as `&#039;`. 

The `htmlentities` method shows the text as it is, I suppose it uses a different way to encode the text, I would like to research the laravel directive to say it for sure. 

Quick note: the `htmlentities` method is not really necessary to fix the specific encoding issue I added it to make it secure for potential future XSS issues (as far as I can tell currently there is no way to the user to modify the metadata but this is a shared library)